File: tests/indexing/parsing/lang/fixtures/typescript/BodySizes.ts
const square = (n: number): number => n * n;

function noop(): void {}

async function loadAll(ids: number[], limit: number): Promise<Item[]> {
  const results: Item[] = [];
  for (const id of ids) {
    if (results.length >= limit) {
      break;
    }
    const item = await fetch(`/items/${id}`);
    results.push(await item.json());
  }
  return results;
}

File: tests/indexing/parsing/lang/fixtures/cpp/BodySizes.cpp
int forward_declared(int a);

int empty_body() {}

int accumulate(const std::vector<int> &items, int limit) {
    int result = 0;
    for (int value : items) {
        if (result >= limit) {
            break;
        }
        result += value;
    }
    return result;
}
